src/app-logic/published-profiles-store.js

```diff
@@ -57,15 +59,30 @@ export type ProfileData = {|
 // Exported for tests.
 export const DATABASE_NAME = 'published-profiles-store';
 export const OBJECTSTORE_NAME = 'published-profiles';
-export const DATABASE_VERSION = 1;
+export const DATABASE_VERSION = 2;
 
 async function reallyOpen(): Promise<Database> {
   const db = await openDB(DATABASE_NAME, DATABASE_VERSION, {
-    upgrade(db) {
-      const store = db.createObjectStore(OBJECTSTORE_NAME, {
-        keyPath: 'profileToken',
-      });
-      store.createIndex('originHostname', 'originHostname');
+    upgrade(db, oldVersion, newVersion, transaction) {
+      switch (oldVersion) {
+        case 0: {
+          // Version 1: this is the first version of the DB.
+          const store = db.createObjectStore(OBJECTSTORE_NAME, {
+            keyPath: 'profileToken',
+          });
+          store.createIndex('originHostname', 'originHostname');
+        }
+        /* fallthrough */
```

Comment:
I don't understand the fallthrough here.

Reply:
ah, this comes from this eslint rule: https://eslint.org/docs/rules/no-fallthrough#disallow-case-statement-fallthrough-no-fallthrough: `It's considered a best practice to always indicate when a fallthrough is intentional using a comment which matches the /falls?\s?through/i regular expression:`
I think a longer comment could match this regexp too, I'll make it more explicit.

Reply:
I decided to disable the rule for the full switch block, with a comment at the top. Thanks for the feedback!